experimentation/tools/README.md

```diff
@@ -63,7 +147,7 @@ with statistics collection.
 The command executed above creates a file `prs.json` with an initial record
 like this:
 
-```
+```json
```

Comment:
I think there is a " that should be removed in line 158, where "merged_at" is.

Reply:
Yep, wonder how that got there. Removed.